js/index.js

```diff
@@ -226,14 +232,19 @@
           .append("a")
             .attr("target", "_blank")
             .attr("title", function(d) {
-              return d.page_title;
+              return d.page_title + " (" + domainFromPage(d.page) + ")";
             })
             .attr("href", function(d) {
               return exceptions[d.page] || ("http://" + d.page);
             })
             .text(function(d) {
-              return title_exceptions[d.page] || d.page_title;
-            });
+              return (title_exceptions[d.page] || d.page_title) + " ";
```

Comment:
I think you could avoid putting in the `" "` by giving `span.domain` an attribute of `display: inline-block` and then using `padding` around it.